README.md

```diff
@@ -106,7 +122,6 @@ processGltf(gltf, options)
 |`--json`, `-j`|Convert the input glb to glTF.|No, default `false`|
 |`--separate`, `-s`|Write separate buffers, shaders, and textures instead of embedding them in the glTF.|No, default `false`|
 |`--separateTextures`, `-t`|Write out separate textures only.|No, default `false`|
-|`--checkTransparency`|Do a more exhaustive check for texture transparency by looking at the alpha channel of each pixel. By default textures are considered to be opaque.|No, default `false`|
 |`--stats`|Print statistics to console for input and output glTF files.|No, default `false`|
```

Comment:
When running `--stats`, the reported input `Images`, `Nodes`, `Meshes`, `Materials`, and `Animations` are all undefined. Output stats are fine.

Reply:
It was failing for glTF 1.0 models.

I changed the code and doc to only print stats once for the final model.